tidy3d/components/data/monitor_data.py

```diff
@@ -65,28 +68,24 @@ def grid_locations(self) -> Dict[str, str]:
     def symmetry_eigenvalues(self) -> Dict[str, Callable[[Axis], float]]:
         """Maps field components to their (positive) symmetry eigenvalues."""
 
-    def apply_symmetry(self, simulation: Simulation) -> AbstractFieldData:
-        """Return copy of self with symmetry applied."""
-        return self._apply_field_symmetry(
-            symmetry=simulation.symmetry,
-            symmetry_center=simulation.center,
-            grid_expanded=simulation.discretize(self.monitor, extend=True),
-        )
-
-    def _apply_field_symmetry(  # pylint:disable=too-many-locals
+    def apply_symmetry(  # pylint:disable=too-many-locals
         self,
         symmetry: Tuple[Symmetry, Symmetry, Symmetry],
         symmetry_center: Coordinate,
         grid_expanded: Grid,
     ) -> AbstractFieldData:
-        """Create a copy of the :class:`.AbstractFieldData` with symmetry applied
+        """If symmetry is present, create a copy of the :class:`.AbstractFieldData` with the fields
+        expanded. If symmetry not present, returns the same data object.
 
         Returns
         -------
         :class:`AbstractFieldData`
-            A new data object with the symmetry expanded fields.
+            A data object with the symmetry expanded fields.
         """
 
+        if np.all(symmetry == 0):
```

Comment:
This will always evaluate to `False` because `symmetry` is a tuple.  I think instead we want
```py
if all(s==0 for s in symmetry)
```